crates/oxc_linter/src/snapshots/no_jsx_as_prop.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_jsx_as_prop
+---
+  ⚠ eslint-plugin-react-perf(no-jsx-as-prop):
```

Comment:
Can you finish the error messages and help.

Reply:
It's already changed. It's in the new snapshot file. I'll fix it.